packages/hardhat-toolbox/package.json

```diff
@@ -73,6 +75,8 @@
   "peerDependencies": {
     "@nomicfoundation/hardhat-chai-matchers": "^2.0.0",
     "@nomicfoundation/hardhat-ethers": "^3.0.0",
+    "@nomicfoundation/hardhat-ignition": "^0.13.0",
+    "@nomicfoundation/hardhat-ignition-ethers": "^0.13.0",
```

Comment:
In the readme it notes that network helpers have to be re-exported from the toolbox because IDEs don't pick up the peer dependency install.

Do we need to do the same for `const { buildModule } = require("@nomicfoundation/hardhat-ignition/modules");`?

Reply:
We have opted to not re-export for the moment. If this proves an actual issue to users (how often is intellisense for imports in Ignition modules an issue?), we will reconsider.